util/energy_saving_calculation.py

```diff
@@ -5,9 +5,11 @@
 from dateutil import parser
 import redis
 import pytz
+from sqlalchemy.sql import func
 from models.user import User
+from models.loadprofile import LoadProfileEntry
 from util.error import exception_message
-from util.database import get_engine
+from util.database import get_engine, create_session
```

Comment:
Siehe oben - beides brauchen wir hier nicht. Es kann übrigens auch leicht sein, dass `create_session()` Dir dann eine zusätzliche Datenbank in einem Unterverzeichnis anlegt. Da einfach mal nachschauen, es darf nur `mybuzzn.db` im Root-Verzeichnis geben. Alles andere soll in-memory passieren.